dagmc_stats/dagmc_stats.py

```diff
@@ -210,15 +227,20 @@ def get_tri_side_length(my_core, tri):
         coord_list.append(coords)
 
     for side in range(3):
-        side_lengths.append(np.linalg.norm(coord_list[side]-coord_list[side-2]))
-        # The indices of coord_list includes the "-2" because this way each side will be matched up with both
-        # other sides of the triangle (IDs: (Side 0, Side 1), (Side 1, Side 2), (Side 2, Side 0))
+        side_lengths.update({verts[side-1]:
+                             np.linalg.norm(coord_list[side] -
+                             coord_list[side-2])})
+        # The indiex of vert corresponds with the vert opposite to the side
+        # The indices of coord_list includes the "-2" because this way each
+        # side will be matched up with both other sides of the triangle
+        # (IDs: (Side 0, Side 1), (Side 1, Side 2), (Side 2, Side 0))
```

Comment:
Since I'm the one most anxious about understanding this in the future, I'll make a specific recommendation on this comment.
```suggestion
        # Although it may not be intuitive, the indexing of these lists takes 
        # advantage of python's indexing syntax to rotate through 
        # the `verts` of the triangle while simultaneously referencing the side
        # opposite each of the `verts` by the coordinates of the vertices that
        # define that side:
        #    side       side-1   index(side-1)     side-2   index(side-2)
        #     0               -1           2                      -2             1
        #     1                0           0                      -1             2
        #     2                1           1                       0             0
```